region/crateria/east.json

```diff
@@ -972,9 +973,17 @@
           "sparking": {
             "runways": [
               {
-                "length": 28,
+                "length": 4,
+                "steepUpTiles": 1,
                 "requires": null,
                 "openEnd": 0
+              },
+              {
+                "length": 28,
+                "steepUpTiles": 5,
+                "steepDownTiles": 1,
+                "requires": ["Gravity"],
+                "openEnd": 0
```

Comment:
This one also needs the usableComingIn false because of Sciser hits

Reply:
I neglected to actually check that room, I think. I'll go take a look and adjust.